### PR TITLE
Removed the pinning of stomp.py to <7.0.0 to pick up fixes

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

### CODE REVIEW

File: tests/unit/zhmcclient/test_notification.py [hunk 1]
```diff
@@ -23,14 +23,38 @@
...
 
 
+def create_event_args(headers, message):
+    """
+    Transform headers, message to event method parameters.
+    This is the inverse of _NotificationListener.get_headers_message().
+    """
+    # Indicator whether stomp uses Frame objects for the event listener
+    # methods. stomp.py introduced this in version 7.0.0, but since it
+    # changed its __version__ attribute from tuple to string in version
```
Comment (EdGue42) [inline tests/unit/zhmcclient/test_notification.py:43]: Out of curiosity: isn't using a tuple for version "better" than a flat string?

File: zhmcclient/_auto_updater.py [hunk 2]
```diff
@@ -227,6 +236,58 @@ def __init__(self, updater, session):
         self._session = session
         self._client = None
 
+        # Lazy importing of the stomp module, because the import is slow in some
+        # versions.
+        # pylint: disable=import-outside-toplevel
+        import stomp
+        self._stomp = stomp
+
+        # Indicator whether stomp uses Frame objects for the event listener
```
Comment (EdGue42) [inline zhmcclient/_auto_updater.py:245]: comment+code duplication. could this go into a helper method?

File: zhmcclient/_notification.py [hunk 3]
```diff
@@ -434,6 +440,57 @@ def __init__(self, handover_dict, handover_cond):
...
+        import stomp
+        self._stomp = stomp
+
+        # Indicator whether stomp uses Frame objects for the event listener
+        # methods. stomp.py introduced this in version 7.0.0, but since it
+        # changed its __version__ attribute from tuple to string in version
+        # 8.1.1, it would be fairly complex to check for version 7.0.0.
+        # Instead, we utilize the fact that we have either versions <5.0 or
+        # >8.1.1 and thus can test for the __version__ type.
+        self._stomp_uses_frames = isinstance(self._stomp.__version__, str)
```
Comment (EdGue42) [inline zhmcclient/_notification.py:455]: one more.

**Review by EdGue42 — APPROVED** [review]

File: tests/unit/zhmcclient/test_notification.py (hunk 1 above)
Comment (andy-maier) [reply tests/unit/zhmcclient/test_notification.py:43]: By convention, `__version__` contains the string, and `__version_info__` or `version_info` the tuple.
https://stackoverflow.com/a/459185/1424462
https://peps.python.org/pep-0396/

File: zhmcclient/_auto_updater.py (hunk 2 above)
Comment (andy-maier) [reply zhmcclient/_auto_updater.py]: done.

File: zhmcclient/_notification.py (hunk 3 above)
Comment (andy-maier) [reply zhmcclient/_notification.py]: done.

**Pull request merged**